### PR TITLE
Update Hero.jsx

### PR DESCRIPTION
**Pull request by Aravindh020**

### CODE REVIEW

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -26,10 +26,10 @@ class Hero extends Component {
...
                 {/* social */}
-                <a href="/" className="github"><img alt="" src="https://camo.githubusercontent.com/ce676e05faf5d1d3cfa8452e3bc556555809eb74/68747470733a2f2f63646e2e737667706f726e2e636f6d2f6c6f676f732f6769746875622d69636f6e2e737667" height="35" width="35" ></img></a>
-                <a href="/" className="instagram"><img alt="" src="https://github.com/TheDudeThatCode/TheDudeThatCode/raw/master/Assets/Instagram.svg" height="35" width="35" ></img></a>
-                <a href="/" className="linkedin"><img alt="" src="https://danielletaylordotcom.files.wordpress.com/2015/12/icon-linkedin.png" height="35" width="35" ></img></a>
-                <a href="/" className="facebook"><img alt="" src="https://cdn4.iconfinder.com/data/icons/social-messaging-ui-color-shapes-2-free/128/social-facebook-circle-64.png" height="35" width="35" ></img></a>
+                <a href="https://www.facebook.com/dscwow" className="github"><img alt="FB" src="https://camo.githubusercontent.com/ce676e05faf5d1d3cfa8452e3bc556555809eb74/68747470733a2f2f63646e2e737667706f726e2e636f6d2f6c6f676f732f6769746875622d69636f6e2e737667" height="35" width="35" ></img></a>
```
Comment (aravindvnair99) [inline src/components/Hero.jsx:29]:
```suggestion
                <a href="https://facebook.com/dscwow" className="facebook"><img alt="Facebook" src="https://camo.githubusercontent.com/ce676e05faf5d1d3cfa8452e3bc556555809eb74/68747470733a2f2f63646e2e737667706f726e2e636f6d2f6c6f676f732f6769746875622d69636f6e2e737667" height="35" width="35" ></img></a>
```

File: src/components/Hero.jsx [hunk 2]
```diff
@@ -26,10 +26,10 @@ class Hero extends Component {
...
+                <a href="https://www.instagram.com/dsc.wow/" className="instagram"><img alt="" src="https://github.com/TheDudeThatCode/TheDudeThatCode/raw/master/Assets/Instagram.svg" height="35" width="35" ></img></a>
```
Comment (aravindvnair99) [inline src/components/Hero.jsx:30]:
```suggestion
                <a href="https://instagram.com/dsc.wow/" className="instagram"><img alt="Instagram" src="https://github.com/TheDudeThatCode/TheDudeThatCode/raw/master/Assets/Instagram.svg" height="35" width="35" ></img></a>
```

File: src/components/Hero.jsx [hunk 3]
```diff
@@ -26,10 +26,10 @@ class Hero extends Component {
...
+                <a href="https://www.linkedin.com/company/dsc-wow/" className="linkedin"><img alt="" src="https://danielletaylordotcom.files.wordpress.com/2015/12/icon-linkedin.png" height="35" width="35" ></img></a>
```
Comment (aravindvnair99) [inline src/components/Hero.jsx:31]:
```suggestion
                <a href="https://linkedin.com/company/dsc-wow/" className="linkedin"><img alt="LinkedIn" src="https://danielletaylordotcom.files.wordpress.com/2015/12/icon-linkedin.png" height="35" width="35" ></img></a>
```

File: src/components/Hero.jsx [hunk 4]
```diff
@@ -26,10 +26,10 @@ class Hero extends Component {
...
+                <a href="https://twitter.com/DscWow/" className="facebook"><img alt="" src="https://cdn4.iconfinder.com/data/icons/social-messaging-ui-color-shapes-2-free/128/social-facebook-circle-64.png" height="35" width="35" ></img></a>
```
Comment (aravindvnair99) [inline src/components/Hero.jsx:32]:
```suggestion
                <a href="https://twitter.com/DscWow" className="twitter"><img alt="Twitter" src="https://cdn4.iconfinder.com/data/icons/social-messaging-ui-color-shapes-2-free/128/social-facebook-circle-64.png" height="35" width="35" ></img></a>
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Aravindh020 The class names are wrong and a few are missing the `alt` attribute.

**Comment by Aravindh020** [comment]
Source was blank, I just added links. Didn't make any change in other part

File: src/components/Hero.jsx [hunk 5]
```diff
@@ -26,10 +26,10 @@ class Hero extends Component {
...
                 {/* social */}
-                <a href="/" className="github"><img alt="" src="https://camo.githubusercontent.com/ce676e05faf5d1d3cfa8452e3bc556555809eb74/68747470733a2f2f63646e2e737667706f726e2e636f6d2f6c6f676f732f6769746875622d69636f6e2e737667" height="35" width="35" ></img></a>
-                <a href="/" className="instagram"><img alt="" src="https://github.com/TheDudeThatCode/TheDudeThatCode/raw/master/Assets/Instagram.svg" height="35" width="35" ></img></a>
-                <a href="/" className="linkedin"><img alt="" src="https://danielletaylordotcom.files.wordpress.com/2015/12/icon-linkedin.png" height="35" width="35" ></img></a>
-                <a href="/" className="facebook"><img alt="" src="https://cdn4.iconfinder.com/data/icons/social-messaging-ui-color-shapes-2-free/128/social-facebook-circle-64.png" height="35" width="35" ></img></a>
+                <a href="https://facebook.com/dscwow" className="facebook"><img alt="Facebook" src="https://camo.githubusercontent.com/ce676e05faf5d1d3cfa8452e3bc556555809eb74/68747470733a2f2f63646e2e737667706f726e2e636f6d2f6c6f676f732f6769746875622d69636f6e2e737667" height="35" width="35" ></img></a>
```
Comment (aravindvnair99) [inline src/components/Hero.jsx:29]:
```suggestion
                <a href="https://facebook.com/dscwow" className="facebook"><img alt="Facebook" src="https://cdn4.iconfinder.com/data/icons/social-messaging-ui-color-shapes-2-free/128/social-facebook-circle-64.png" height="35" width="35" ></img></a>
```

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**